near/nearcore
# Add an orphan section to chain info debug page

### Description
This PR adds an orphan section to the chain info debug page that displays hash and height info of orphans.

### JIRA Issue
[CP-22](https://nearinc.atlassian.net/browse/CP-22)

### Test plan

 - Build with make neard
 - Under `nearcore` directory, launch localnet using command `nearup stop && nearup run localnet --binary-path ./target/release`
 - Open your browser and go to http://[host_address]:3030/debug/chain_info
 - Expect to see a webpage that looks similar to the one below
(Positive case TBD)

### Screenshot on localnet
<img width="1509" alt="Screen Shot 2022-04-04 at 4 22 52 PM" src="https://user-images.githubusercontent.com/98097537/161648476-bb575d82-1486-47c2-859a-2878625497b0.png">

### First round reviewer
 - @mzhangmzz 
 - @mm-near
